tekton/resources/images/docker-multi-arch-template.yaml

```diff
@@ -89,8 +89,9 @@ spec:
             for tag in ${TAGS}; do DESTINATIONS="${DESTINATIONS} --tag ${IMAGE_URL}:${tag}"; done
             IFS=$OIFS
 
-            gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-            gcloud auth configure-docker
+            IMAGE_REGISTRY_USER=tekton-robot
+            IMAGE_REGISTRY_TOKEN=$(cat "$IMAGE_REGISTRY_CREDENTIALS")
+            docker login -u ${IMAGE_REGISTRY_USER} --password-stdin ${IMAGE_REGISTRY_TOKEN}            
```

Comment:
Same feedback as on https://github.com/tektoncd/plumbing/pull/2327 re. correct use of `--password-stdin`